app/org/sagebionetworks/bridge/dynamodb/DynamoSubpopulationDao.java

```diff
@@ -113,7 +131,13 @@ public Subpopulation createDefaultSubpopulation(StudyIdentifier studyId) {
         subpop.setMinAppVersion(0);
         subpop.setDefaultGroup(true);
         // The first group is required until the study designers say otherwise
-        subpop.setRequired(true); 
+        subpop.setRequired(true);
+        
+        // We know in this case that criteria do not exist
+        Criteria criteria = criteriaDao.copyCriteria(getKey(subpop), subpop);
+        criteriaDao.createOrUpdateCriteria(criteria);
+        subpop.setCriteria(criteria);
+        
```

Comment:
Instead of adding the criteria to the Subpop and then copying it to the criteria, it would be cleaner to create the criteria directly.